### ISSUE TITLE
Fix the text size and padding when the `Dropdown` box is empty with `multiselect=True`

### ISSUE THREAD

**Issue by abidlabs** [gradio-app/gradio#3163]
Created an issue so that we keep track of this @dawoodkhan82 

Originally noted by @aliabid94: the text size and padding when the box is empty look a little off to me:
![Recording 2023-02-07 at 13 44 05](https://user-images.githubusercontent.com/7870876/217349363-409c568d-113d-441d-afb9-64a3a7ee8d59.gif)

if you wanna just patch in this PR, or maybe tackle later

_Originally posted by @aliabid94 in https://github.com/gradio-app/gradio/issues/3145#issuecomment-1421351216_

**Comment by Neilblaze** [comment]
@abidlabs I'd love to take a shot if it's up for grabs :)

**Comment by abidlabs** [comment]
Thanks @Neilblaze! @dawoodkhan82 have you already started working on this or is this up for grabs?

**Comment by dawoodkhan82** [comment]
It's up for grabs, I think this would be a great first issue. Thanks @Neilblaze

**Comment by Neilblaze** [comment]
Thanks @dawoodkhan82! 
I was exploring the codebase and what I understood is that an update for `padding` & `font-size` in [this](https://github.com/Neilblaze/gradio/blob/main/ui/packages/form/src/Dropdown.svelte#L47&&#L50) is the task, right?

Do correct me if my approach is wrong. 😅

**Comment by dawoodkhan82** [comment]
@Neilblaze It would actually be the updating the padding and width in the `Multiselect.svelte` file.

**Comment by Neilblaze** [comment]
Thanks for the help! @dawoodkhan82 😄 
I did try to run it locally, but I'm getting into [this](https://github.com/gradio-app/gradio/issues/2686) issue everytime I run `bash scripts/build_frontend.sh` in the root. I've followed all instructions very carefully & tried it on codespaces (after I failed to run it on my local), and even downgrading my default node (19.x.x) to `node --version 16.x.x`, but nothing is fixing the issue.

**Comment by abidlabs** [comment]
Just to confirm, did you try setting this environmental variable?

> @libai-lab Can you add the environment variable NODE_OPTIONS="--max-old-space-size=4096" like we do i[n our CI](https://github.com/gradio-app/gradio/blob/main/.github/workflows/ui.yml#L16)?

**Comment by Neilblaze** [comment]
@abidlabs yep, I've tried that. Nothing works as of now.

**Comment by Neilblaze** [comment]
@abidlabs I retried starting from scratch, and I'm getting the same issue again. Here's a snapshot of the err log when I run `bash scripts/build_frontend.sh` in the root.

> P.S. → I did add the environment variable `NODE_OPTIONS="--max-old-space-size=4096"`

![image](https://user-images.githubusercontent.com/48355572/219497105-00e4294d-2ccd-418c-b192-2581145d53df.png)

**Comment by abidlabs** [comment]
And what is the `npm` and `pnpm` versions in this environment?

**Comment by Neilblaze** [comment]
> And what is the `npm` and `pnpm` versions in this environment?

npm version is `9.3.1` and pnpm is `7`

**Comment by abidlabs** [comment]
I'm stumped, all that looks right to me. @pngwn would you have any other recommendations?

**Comment by pngwn** [comment]
@Neilblaze Could you run `npx envinfo --system --binaries` or `pnpx envinfo --system --binaries` and paste the output. Also post your full `pnpm` version: `pnpm -v`.

**Comment by Neilblaze** [comment]
@pngwn Apologies for the delay, here you go:

- **envinfo** :

```sh
  System:
    OS: Linux 5.4 Ubuntu 20.04.5 LTS (Focal Fossa)
    CPU: (2) x64 Intel(R) Xeon(R) Platinum 8272CL CPU @ 2.60GHz
    Memory: 2.38 GB / 3.84 GB
    Container: Yes
    Shell: 5.0.17 - /bin/bash
  Binaries:
    Node: 19.6.0 - ~/nvm/current/bin/node
    Yarn: 1.22.19 - /usr/bin/yarn
    npm: 9.4.0 - ~/nvm/current/bin/npm
```

- **pnpm** version : `7.27.1`

> P.S. :> I'm running on GitHub codespaces.

**Comment by abidlabs** [comment]
Closed via #3211
